src/main/java/net/sf/jabref/gui/keyboard/KeyBindingWrapper.java

```diff
@@ -0,0 +1,50 @@
+package net.sf.jabref.gui.keyboard;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import org.apache.commons.lang.WordUtils;
+
+public class KeyBindingWrapper {
+
+    private KeyBinding keyBinding;
+    private final SimpleStringProperty bind;
+    private final KeyBindingCategory category;
+
+
+    public KeyBindingWrapper(KeyBinding keyBinding, String bind, KeyBindingCategory category) {
```

Comment:
In the best case you just need the `KeyBinding` parameter and get/set the rest from it.